artwork-data/artworks-overrides.json

```diff
@@ -2665,7 +2665,7 @@
       "country": "United States",
       "postalCode": "95113",
       "state": "CA",
-      "sourceURL": "https://www.powwowsanjose.com/"
+      "sourceURL": "https://www.sanjoseca.gov/home/showdocument?id=32425"
```

Comment:
Please revert these changes. PowWowSanJose.com murals and public art data are not the same as the City of San Jose public art data. These should still be pointing to PowWowSanJose.com

Reply:
The only powwowsanjose links that got changed are dead links, so I'm a little confused. Are you suggesting that all dead powwowsanjose links should just be changed to www.powwowsanjose.com ? Or just leave them out (change to blank)? Thanks.

Reply:
These links should not be pointing to the City of San Jose website. That's not where I got my data from. Also the City of San Jose links to public art that is not the same as the murals that are part of the Pow Wow San Jose festival. 

It looks like all the mural location information is now being posted to https://www.powwowsanjose.com/mural-locations